### DISCUSSION TITLE
v1.5

### DISCUSSION THREAD

**Discussion by MrMeowzz** [MrMeowzz/Funkin-MrMeowzz#44] [Updates]
- fixed iphone chart
- fixed highscores on b-side
- added song speed modifier (not available on web version)
- added keybind support (in settings)
- added note timing option
- pressing 7 in freeplay now opens chart editor on selected song
- improved controller support

<hr /><em>This discussion was created from the release <a href='https://github.com/MrMeowzz/Funkin-MrMeowzz/releases/tag/v1.5'>v1.5</a>.</em>

**Comment by DeceptedPar** [comment]
I love it!
